pkg/bootstrapper/bootstrapper.go

```diff
@@ -182,9 +182,12 @@ func NewWinNodeBootstrapper(k8sInstallDir, ignitionFile, kubeletPath string, cni
 	var dependents []*mgr.Service
 	// If there is already a kubelet service running, find it
 	if ksvc, err := svcMgr.OpenService(KubeletServiceName); err == nil {
-		if dependentSvc, err := svcMgr.OpenService(kubeletDependentSvc); err == nil {
-			dependents = append(dependents, dependentSvc)
+		dependentSvc, err := svcMgr.OpenService(kubeletDependentSvc)
+		if err != nil {
```

Comment:
Correct me if I am wrong but isn't this function called on intialize kubelet as well and when the hybrid-overlay is not yet installed? Won't that always result in an error being returned here?

Reply:
You might be right. How do we handle errors returned by `OpenService` when the dependent services are installed ?

Reply:
we might have to check by ignoring a specific error here.

Reply:
sorry I missed this, but this is only called when kubelet service is already running and not when we are creating it for the first time. Thinking of this scenario I think its fine checking for any error here.

Reply:
Ignoring `service does not exist` 

> we might have to check by ignoring a specific error here.